src/View/SSTemplateParser.peg

```diff
@@ -1156,7 +1168,9 @@ class SSTemplateParser extends Parser implements TemplateParser
 
     # This is used to remove template comments
 
-    Comment: "<%--" (!"--%>" /(?s)./)+ "--%>"
+    CommentWithContent: '<%--' ( !"--%>" /(?s)./ )+ '--%>'
+    EmptyComment: '<%----%>'
```

Comment:
I tried _a lot_ of different things to get this to work as just one rule, but failed :(